tests/testthat/Rmd/front-matter.Rmd

```diff
@@ -0,0 +1,10 @@
+---
+title: "title1"
+confluence_settings:
+  space_key: "space1"
```

Comment:
`spaceKey` or `space_key`....

Reply:
Context: I used `parent_id` for where the actual API param is `ancestors` because it seems very counter-intuitive. My implicit rule is probably something like this:

* Use actual parameter names as it is in simple bindings (e.g. `confl_post_page()`)
* Use user-friendly names in advanced functions (e.g. `confl_create_post_from_Rmd()`)
    * In addition, use snake case instead of camel case

`spaceKey` is not so user-unfriendly, so I was confused between these two. Probably I should use `space_key`.